process/block/metablock.go

```diff
@@ -1842,7 +1813,8 @@ func (mp *metaProcessor) MarshalizedDataToBroadcast(
 
 	bodies := make(map[uint32]block.MiniBlockSlice)
 	for _, miniBlock := range body.MiniBlocks {
-		if miniBlock.ReceiverShardID == mp.shardCoordinator.SelfId() {
+		if miniBlock.SenderShardID != mp.shardCoordinator.SelfId() ||
```

Comment:
Now there is no broadcast for any miniblock. First clause is not needed.

Reply:
It is ok, we want to broadcast only miniblocks created in self shard with destination in other shards